graphene/src/document.rs

```diff
@@ -100,7 +100,7 @@ impl Document {
 	}
 
 	/// Returns a mutable reference to the layer or folder at the path.
-	fn layer_mut(&mut self, path: &[LayerId]) -> Result<&mut Layer, DocumentError> {
+	pub fn layer_mut<'a>(&'a mut self, path: &'a [LayerId]) -> Result<&'a mut Layer, DocumentError> {
```

Comment:
Why do we need the explicit lifetimes and the path to live as long as the resulting layer? (removing the explicit lifetimes compiles fine)

Reply:
Yep, likely residual, removing is fine.